src/hooks/useClientStore.js

```diff
@@ -2,14 +2,20 @@ import create from 'zustand';
 import {persist} from 'zustand/middleware';
 
 const useClientStore = create(
-	persist(set => ({
-		clients: [],
-		addClient: client => {
-			set(state => {
-				return {clients: [...state.clients, client]};
-			});
-		},
-	}))
+	persist(
+		set => ({
+			clients: [],
+			addClient: client => {
+				set(state => {
+					return {clients: [...state.clients, client]};
+				});
+			},
+		}),
+		{
+			name: 'Clients', // unique name
+			getStorage: () => localStorage, // (optional) by default, 'localStorage' is used
+		}
+	)
```

Comment:
As Alex mentioned in our Daily this morning and in the discussion channel: 
there is a "bone to pick" 🍗  between `next.js` and the `Persist middleware` of Zustand 
**Long story short:** 
I would suggest you to work without persist until your [hydration error](https://github.com/orgs/neuefische/discussions/68) is solved - by Alex, because I can't  😁